### PR TITLE
AMLS-4671 | Updated content in Responsible People

### PR DESCRIPTION
**Pull request by owen-opencast**
Updated content in Responsible People - removed contractions, etc.

## Related / Dependant PRs?

Acceptance test changes [here](https://github.com/hmrc/amls-acceptance-test/pull/163).

## How Has This Been Tested?

Passes spec tests and acceptance tests.

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -430,11 +430,11 @@ lbl.na = Not applicable
 lbl.day = Day
 lbl.month = Month
 lbl.year = Year
-lbl.date.example = For example, 31 03 2017.
+lbl.date.example = For example, 31 03 2017
```
Comment (slavvco) [inline conf/messages]: this was recently changed, requested in AMLS-4656, looks like we are changing this back and forth :/

**Comment by owen-opencast** [comment]
I'll check the comments with @lisadunn577 when she gets back

File: conf/messages [hunk 2]
```diff
@@ -69,8 +69,8 @@ error.invalid.maxlength.1200 = You can’t enter more than 1200 characters
 
 error.survey.satisfaction.required = Select one option to show how you feel about using the service
 
-error.required.phone.number = Enter a telephone number, like 01632 960 001, 07700 900 982 or +44 0808 157 0192. 
-err.invalid.phone.number = Enter a valid contact phone number
+error.required.phone.number = Enter a telephone number, like 01632 960 001, 07700 900 982 or +44 0808 157 0192 (no full stops for error messages)
```
[the remark this answers is not in the document]
Comment (owen-opencast) [reply conf/messages]: Removed

File: conf/messages [hunk 3]
```diff
@@ -69,8 +69,8 @@ error.invalid.maxlength.1200 = You can’t enter more than 1200 characters
...
+err.invalid.phone.number = Enter a telephone number, like 01632 960 001, 07700 900 982 or +44 0808 157 0192 (no full stops for error messages)
```
[the remark this answers is not in the document]
Comment (owen-opencast) [reply conf/messages]: Removed

File: conf/messages (hunk 1 above)
Comment (owen-opencast) [reply conf/messages]: Changed, full stop is to be kept.

**Pull request merged**